src/Core/AdminConsole/Models/Data/Integrations/IntegrationHandlerResult.cs

```diff
@@ -0,0 +1,16 @@
+﻿namespace Bit.Core.Models.Data.Integrations;
```

Comment:
Namespace does not match the folder structure - looks like this is the case for most files here.